unlock-app/src/__tests__/components/lock/ShowUnlessUserHasKeyToAnyLock.test.js

```diff
@@ -1,11 +1,12 @@
 import React from 'react'
-import { shallow } from 'enzyme'
+import * as rtl from 'react-testing-library'
+import 'jest-dom/extend-expect'
 
 import {ShowUnlessUserHasKeyToAnyLock} from '../../../components/lock/ShowUnlessUserHasKeyToAnyLock'
 
 describe('ShowUnlessUserHasKeyToAnyLock', () => {
 
-  it('should show the children if there is no key for this lock', () => {
+  test('should show the children if there is no key for this lock', () => {
```

Comment:
We used `it` all over. Is that a problem? For consistency I suggest we stay with `it`.

Reply:
done